polybar-scripts/popup-calendar/popup-calendar.sh

```diff
@@ -1,27 +1,36 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Please use `#!/bin/sh` and fix all shellcheck errors.

Reply:
With `#!/bin/sh` shellcheck reports warnings [SC2039](https://github.com/koalaman/shellcheck/wiki/SC2039) in lines 14, 16, 23. But it works properly on my system. Should i try to fix it or `#!/bin/sh` will be enough?

Reply:
Yes, please fix the errors. 
I guess it fails at your conditions. Try to use `if [ .. ]; then` instead of `if (( .. )); then`.